content/graphics-switch-ubuntu.md

```diff
@@ -102,6 +102,8 @@ Now enable the System76 Power extension with the toggle to the right.
 Click the system menu in the top right corner of your screen to access graphics
 switching.
 
+>**Note**: Ubuntu versions higher then 22.04 will not have this menu as the GNOME Shell extension has not been updated for newer GNOME versions.
```

Comment:
"higher then" should be "higher than"

Reply:
I'd recommend referencing (and linking to, if possible) the command-line section within this note to present the alternative to what we're saying isn't an option.

I'd also replace "as" with "because".